src/components/GlobalPreferences/CustomLabels/ShareModal.js

```diff
@@ -88,7 +88,7 @@ function ShareModal({ inputRef, onClose, onCopy, onFocus, link, visible }) {
           />
         </div>
       </main>
-      <footer
+      <div
```

Comment:
Why did you choose to remove the `footer` element here? We could' ve added the `header` to make the Modal content semantic no?

Reply:
Mostly because I don't see the main action buttons in this section as belonging in a `<footer />` but that's just my feelings since the element's commonly used as ["author or license text"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer).